core/src/dird/vbackup.cc

```diff
@@ -452,6 +452,22 @@ void NativeVbackupCleanup(JobControlRecord* jcr, int TermCode, int JobLevel)
     jcr->setJobStatusWithPriorityCheck(JS_ErrorTerminated);
   }
 
+  if (jcr->dir_impl->vf_jobids && jcr->dir_impl->vf_jobids[0] != '\0') {
+    using namespace std::string_literals;
+    Jmsg(jcr, M_INFO, 0,
+         "Replicating deleted files from jobids %s to jobid %d\n",
+         jcr->dir_impl->vf_jobids, jcr->JobId);
+    PoolMem q1(PM_MESSAGE);
+    jcr->db->FillQuery(
+        q1, BareosDbQueryEnum::SQL_QUERY::select_recent_version_with_basejob,
+        jcr->dir_impl->vf_jobids, jcr->dir_impl->vf_jobids,
+        jcr->dir_impl->vf_jobids, jcr->dir_impl->vf_jobids);
+    std::string query
+        = "INSERT INTO File (FileIndex, JobId, PathId, LStat, MD5, Name) SELECT FileIndex, 5 AS JobId, PathId, LStat, MD5, Name FROM ("s
```

Comment:
Typo
```suggestion
        = "INSERT INTO File (FileIndex, JobId, PathId, LStat, MD5, Name) SELECT FileIndex, "s + std::to_string(jcr->JobId) + " AS JobId, PathId, LStat, MD5, Name FROM ("s
```
If desired we could even replace this by just a std::format / fmt::format.